armi/reactor/blueprints/reactorBlueprint.py

```diff
@@ -61,17 +60,20 @@ class SystemBlueprint(yamlize.Object):
         which is in turn included into the overall blueprints within
         :py:class:`~armi.reactor.blueprints.Blueprints`.
 
-        This class includes a :py:meth:`~armi.reactor.blueprints.reactorBlueprint.SystemBlueprint.construct`
-        method, which is typically called from within :py:func:`~armi.reactor.reactors.factory`
-        during the initialization of the reactor object to instantiate the core
-        and/or spent fuel pool objects. During that process, a spatial grid is
-        constructed based on the grid blueprints specified in the "grids" section
-        of the blueprints (see :need:`I_ARMI_BP_GRID`) and the assemblies needed
-        to fill the lattice are built from blueprints using :py:meth:`~armi.reactor.blueprints.Blueprints.constructAssem`.
-
-    .. note:: We use string keys to link grids to objects that use them. This differs
-        from how blocks/assembies are specified, which use YAML anchors. YAML anchors
-        have proven to be problematic and difficult to work with
+        This class includes a
+        :py:meth:`~armi.reactor.blueprints.reactorBlueprint.SystemBlueprint.construct` method, which
```

Comment:
Why is the bottom paragraph in alignment with 120 char but the upper paragraph is still what appears to be 100 char?

Reply:
Okay, I had to look this one up.  I opened it in my IDE and found:

* The paragraph of text under the `.. impl::` block tops out at 81 characters.
* This paragraph that begins with "This class includes" now tops out at 100 characters.

So, I suppose to answer your question, I _could_ fix them both so they both top out at 100 or 120 characters. 

Why didn't I?  Two reasons:

1. I was trying to keep this PR small.  I understand I failed at that, but I did _try_.
2. It's hard to decide where to draw the line.  That `impl` text is not formatted on any particular rule.

Reply:
So ruff ignores it, is that the explanation?